src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/components/ApplicantListItem/index.js

```diff
@@ -36,7 +36,7 @@ const ApplicantListItem = ({
 
   useEffect(() => {
     loadAvatar(profile);
-    createNickname(profile);
+    setHasNickname(profile.FirstName === profile.NickName && profile.NickName !== '');
```

Comment:
```suggestion
    setHasNickname(profile.FirstName !== profile.NickName && profile.NickName !== '');
```

Reply:
Ahh right, good catch. That's what I get for copying it manually instead of cutting and pasting.